models/curvefi/readme.md

```diff
@@ -0,0 +1,23 @@
+# Curvefi Spellbook Models 
+
+Welcome to Curve! Here's a short getting started guide for navigating the protocol, and the model files. 
```

Comment:
@0xBoxer more reference for your framework/process to add readme's -- andrew doing archwizard material here on this one 🧙‍♂️

Reply:
fyi @augustog